src/foapy/characteristics/volume.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+from foapy.intervals import intervals
+from foapy.order import order
+
+
+def volume(X, binding, mode):
```

Comment:
I stand on that initial implementation should be `volume(intervals)`.
```suggestion
def volume(intervals):
```